content/es/become-a-sponsor.md

```diff
@@ -2,7 +2,7 @@
 title: Conviértete en un patrocinador
 slug: become-a-sponsor
 top_graphic: 4
-lastmod: 2019-01-28
+lastmod: 2019-08-29
```

Comment:
The year is `2019`

Reply:
That correct and coherent with the English version, right? https://github.com/letsencrypt/website/blob/main/content/en/become-a-sponsor.md

Reply:
I'm not entirely sure if these should be updated when we're making changes, but I assumed we do.

Reply:
The date must match the date of the translated document. As this PR is a translation for the most up-to-date version, it's correct. The dates are used to display on the top of the document a sentence "this translation is not up-to date, you can see the last version of the document in English [Here]".
When translations are updated from Crowdin (like here), the dates should be correct.